tests/unittests/tests-unicoap/tests-unicoap-options.c

```diff
@@ -0,0 +1,204 @@
+    };
+
+    TEST_ASSERT_EQUAL_INT(sizeof(options_blob), unicoap_options_size(&options));
+    _TEST_ASSERT_EQUAL_BYTES(options_blob, unicoap_options_data(&options), sizeof(options_blob));
+}
+
+static void test_remove_multiple(void)
+{
+    UNICOAP_OPTIONS_ALLOC(options, 2100);
+    TEST_ASSERT_EQUAL_INT(unicoap_options_add(&options, 50, (uint8_t*)poem, _UINT4_MAX + 1), 0);
+    TEST_ASSERT_EQUAL_INT(unicoap_options_add(&options, 12, (uint8_t*)poem, _UINT4_MAX), 0);
+    TEST_ASSERT_EQUAL_INT(unicoap_options_add(&options, 12, (uint8_t*)poem, _UINT4_MAX + 1), 0);
+    TEST_ASSERT_EQUAL_INT(unicoap_options_add(&options, 13, (uint8_t*)poem, _UINT4_MAX + 1), 0);
+    TEST_ASSERT_EQUAL_INT(unicoap_options_set(&options, 8, (uint8_t*)poem, _UINT12_MAX), 0);
+    TEST_ASSERT_EQUAL_INT(unicoap_options_add(&options, 8, (uint8_t*)poem, _UINT12_MAX), 0);
+    TEST_ASSERT_EQUAL_INT(unicoap_options_set(&options, 10, (uint8_t*)poem, 200), 0);
+    TEST_ASSERT_EQUAL_INT(unicoap_options_set(&options, 2, (uint8_t*)poem, _UINT4_MAX + 1), 0);
+    TEST_ASSERT_EQUAL_INT(unicoap_options_set(&options, 70, (uint8_t*)poem, 1), 0);
+    TEST_ASSERT_EQUAL_INT(unicoap_options_set(&options, 1, (uint8_t*)poem, 1), 0);
+
+    TEST_ASSERT_EQUAL_INT(unicoap_options_remove(&options, 12), 0);
+
+    /* options blob, from nanocoap */
+    uint8_t options_blob[] = {
```

Comment:
```suggestion
    static uint8_t options_blob[] = {
```

maybe, to not require a huge stack? same in other functions